scripts/http-vuln-cve2014-3704.nse

```diff
@@ -215,10 +215,12 @@ local function do_sql_query(host, port, uri, user)
       ['Content-Type'] = "application/x-www-form-urlencoded"
     }
   }
-  local res = http.post(host, port, uri .. "/user/login", opt, nil, r)
-  --TODO: Check return status
+  local res = http.post(host, port, uri .. "?q=/user/login", opt, nil, r)
+
+  if string.match(res.body, ">Have you forgotten your password?") then
```

Comment:
@dmiller-nmap , please check the new conditional check, https://github.com/nmap/nmap/pull/905/commits/68786920ac112fcc1bc7372745a584319d55a677. It seems good to me. Please confirm.